test/unit/jobs/test_command_factory.py

```diff
@@ -132,13 +154,13 @@ def __command(self, **extra_kwds):
 
 class MockJobWrapper(object):
 
-    def __init__(self):
+    def __init__(self, job_dir):
         self.write_version_cmd = None
         self.command_line = MOCK_COMMAND_LINE
         self.dependency_shell_commands = []
         self.metadata_line = None
         self.configured_external_metadata_kwds = None
-        self.working_directory = "job1"
+        self.working_directory = job_dir
         self.prepare_input_files_cmds = None
         self.commands_in_new_shell = False
```

Comment:
I set it to True for one test to verify things, the rest of the tests still assume commands are written directly in file. Since the module can do both I think it is fine to test both for now even if the default has changed.